fvtest/gctest/GCConfigTest.cpp

```diff
@@ -37,6 +37,17 @@
 #define STRINGFY(str) DO_STRINGFY(str)
 #define DO_STRINGFY(str) #str
 
+const char *gcTests[] = {"fvtest/gctest/configuration/sample_GC_config.xml",
```

Comment:
because if we pass in the config file in the command line, we will have to parse the config file before calling initGoogle() so we could pass it into  INSTANTIATE_TEST_CASE_P and we don't want to be doing anything before initGoogle() . Or is there another way @ktbriggs ?